volumes/miovision/sql/create-view-volumes_15min_by_class.sql

```diff
@@ -1,13 +1,18 @@
 ﻿CREATE MATERIALIZED VIEW miovision.volumes_15min_by_class AS
-		SELECT 	intersection_uid,
-			class_type,
+		SELECT 	a.intersection_uid,
+			b.class_type_id,
 			dir,
 			leg,
 			datetime_bin,
 			period_type,
 			SUM(volume) AS total_volume
 		FROM miovision.volumes_15min A
 		INNER JOIN miovision.classifications B USING (classification_uid)
-		INNER JOIN miovision.report_dates C USING (class_type, intersection_uid)
-		WHERE C.dt = A.datetime_bin::date
-		GROUP BY intersection_uid, class_type, dir, leg, datetime_bin, period_type
\ No newline at end of file
+		INNER JOIN miovision.report_dates_view C USING (class_type_id, intersection_uid)
+		LEFT OUTER JOIN miovision.exceptions e ON (a.intersection_uid, b.class_type_id) = (e.intersection_uid, e.class_type_id) AND datetime_bin <@ e.excluded_datetime
```

Comment:
@rickl4 do you think `exceptions` is important in the api branch? Or is it more of a thing for the csv dump data?

Reply:
I don't think its important. I guess the only case where it might be useful is if we identify a period where the data is not reliable.

Reply:
I'm moving the csv_data version to a different folder then